src/components/EventDetail.tsx

```diff
@@ -77,34 +59,21 @@ export const EventDetail: React.FC<EventDetailProps> = ({
               </td>
             </tr>
             <tr className="border-b border-low-contrast">
-              <td className="py-2 md:py-4">
-                <span>Trvání</span>
-              </td>
+              <td className="py-2 md:py-4">Sport</td>
               <td className="py-2 font-light md:py-4">
-                {mockEvent.duration} min
+                {sportsLabels[sport] || 'Nezařazeno'}
               </td>
             </tr>
-            <tr className="border-b border-low-contrast">
-              <td className="py-2 md:py-4">Sport</td>
-              <td className="py-2 font-light md:py-4">{sport}</td>
-            </tr>
             <tr className="border-b border-low-contrast">
               <td className="py-2 md:py-4">Pokročilost</td>
-              <td className="py-2 font-light md:py-4">{level}</td>
+              <td className="py-2 font-light md:py-4">{levelLabels[level]}</td>
             </tr>
           </tbody>
         </table>
 
-        {!isUserLoggedIn ? (
-          <button
-            type="submit"
-            className="ml-auto mt-6 whitespace-nowrap rounded-md bg-button px-5 py-2 text-base text-white hover:text-primary"
-          >
-            Zúčastnit se
-          </button>
-        ) : (
+        {isUserLoggedIn && ownerId === userId && (
```

Comment:
Good job! :D Do you need the `isUserLoggedIn` check? There should be no `userId` if the user is not logged in